### DISCUSSION TITLE
Hanzi character

### DISCUSSION THREAD

**Discussion by lilbear4896** [jovibor/HexCtrl#42] [Q&A]
I like your software how display Hanzi character

![Image1](https://user-images.githubusercontent.com/78668760/107133383-62d42c80-68df-11eb-9902-23822c3d2ecb.gif)

**Comment by jovibor** [comment]
Displaying the encoding (locale) of ASCII chars can be set through the menu:
![image](https://user-images.githubusercontent.com/5625787/107133996-c9863400-6939-11eb-98d3-a27d65a39769.png)
or using the `SetEncoding()` method.

**Reply by datasynergyuk** [reply]
I think maybe the OP was asking about how to show (Asian) Unicode characters? This is not supported in the current HexCtrl. It would be very nice if it did this because it would make HexCtrl more much useful. However, I think the problem is that they have non-uniform character widths and this would be a big change to implement?

![image](https://user-images.githubusercontent.com/48149668/107149965-66a19680-6953-11eb-8032-6a2c2e87dc60.png)
